projects/1.16/assets/xplosives-mod/xplosives/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "block.xplosives.arrow_bomb": "弓箭炸弹",
+  "block.xplosives.bonemeal_bomb": "骨粉炸弹",
+  "block.xplosives.cluster_bomb": "集束炸弹",
+  "block.xplosives.fireblast_tnt": "火焰弹TNT",
+  "block.xplosives.hydrogen_bomb": "氢弹",
+  "block.xplosives.incendiary_bomb": "燃烧炸弹",
+  "block.xplosives.lightning_conductor": "雷击炸弹",
+  "block.xplosives.packed_tnt": "压缩TNT",
+  "block.xplosives.physics_bomb": "物理炸弹",
+  "block.xplosives.snowball_bomb": "雪球炸弹",
+  "block.xplosives.tipped_arrow_bomb": "药箭炸弹",
+  "block.xplosives.detonator": "遥控雷管",
+  "block.xplosives.gunpowder_barrel": "火药桶",
+  "block.xplosives.torch_bomb": "火把炸弹",
+  "block.xplosives.soul_torch_bomb": "灵魂火把炸弹",
+  "block.xplosives.mini_tnt": "迷你TNT",
+  "item.xplosives.detonator_remote": "遥控雷管引爆器",
+  "item.xplosives.black_box": "炸弹发射器",
+  "item.xplosives.dynamite": "碰炸炸药",
```

| **审查级别**    |
|--------------|
| ❓疑问 / 需要作者确认 |

没搜到“碰炸炸药”相关信息

原文：dynamite
```suggestion
  "item.xplosives.dynamite": "硝酸甘油炸药",
```
```suggestion
  "item.xplosives.dynamite": "硝化甘油炸药",
```